Source/WebCore/Modules/applepay/ApplePaySession.cpp

```diff
@@ -612,10 +612,10 @@ ExceptionOr<void> ApplePaySession::begin(Document& document)
     if (!canBegin())
         return Exception { ExceptionCode::InvalidAccessError, "Payment session is already active."_s };
 
-    if (paymentCoordinator().hasActiveSession())
+    if (protectedPaymentCoordinator()->hasActiveSession())
         return Exception { ExceptionCode::InvalidAccessError, "Page already has an active payment session."_s };
 
-    if (!paymentCoordinator().beginPaymentSession(document, *this, m_paymentRequest))
+    if (!protectedPaymentCoordinator()->beginPaymentSession(document, *this, m_paymentRequest))
```

Comment:
We should cache it in a local variable instead of calling this function twice. This causes unnecessary ref-counting churn otherwise.